src/main/java/com/synopsys/integration/jira/common/rest/service/PluginManagerService.java

```diff
@@ -191,10 +191,7 @@ private JiraRequest createDeleteRequest(HttpUrl apiUri, String pluginToken) {
     }
 
     private JiraRequest.Builder createBasicRequestBuilder(HttpUrl apiUri) {
```

Comment:
Do you even need this method at this point?

Reply:
I was debating whether this should be removed or not. It was previously adding extra items and maybe it will again in the future? It will be easy to change default requests in the future with this method.

Reply:
I think we should remove this until it is needed. The name of the method no longer applies anyway